crates/turbopack-node/js/src/transforms/webpack-loaders.ts

```diff
@@ -333,7 +333,7 @@ const transform = (
           emitWarning: makeErrorEmitter("warning", ipc),
           emitError: makeErrorEmitter("error", ipc),
           getLogger(name: unknown) {
-            return (type: unknown, args: unknown) => {
+            const logger = (type: unknown, args: unknown) => {
```

Comment:
Would prefer this to not be an arrow function, but it actually uses `this` so it would need additional changes.